src/pages/ExportTransactions/index.tsx

```diff
@@ -0,0 +1,241 @@
+import { DatePicker, InputNumber, Tabs } from 'antd'
+import * as cnLocale from 'antd/es/date-picker/locale/zh_CN'
+import * as enLocale from 'antd/es/date-picker/locale/en_US'
+import dayjs from 'dayjs'
+import { useState } from 'react'
+import classNames from 'classnames'
+import { useTranslation } from 'react-i18next'
+import { useSearchParams, useUpdateSearchParams } from '../../utils/hook'
+import Content from '../../components/Content'
+import styles from './styles.module.scss'
+import 'dayjs/locale/es-us'
+import 'dayjs/locale/zh-cn'
+import { ReactComponent as BlockIcon } from '../../assets/block_icon.svg'
+import { ReactComponent as ErrorIcon } from '../../assets/error_icon.svg'
+import { ReactComponent as SuccessIcon } from '../../assets/success_icon.svg'
+import { exportTransactions } from '../../service/http/fetcher'
+import { omit } from '../../utils/object'
+
+const ExportTransactions = () => {
+  const [t, { language }] = useTranslation()
+
+  const locale = language === 'zh' ? cnLocale.default : enLocale.default
+
+  const {
+    format,
+    tab: tabStr,
+    'start-date': startDateStr,
+    'end-date': endDateStr,
+    'from-height': fromHeightStr,
+    'to-height': toHeightStr,
+  } = useSearchParams('format', 'tab', 'start-date', 'end-date', 'from-height', 'to-height')
+
+  const defaultTab = 'date'
```

Comment:
default value could be assigned to `tabStr`, as `tab: tabStr = 'date'`
And page should be redirected if `tabStr` is invalid